getinsomnia/insomnia
# feat: enable Url in pre-request scripting [INS-3379]

Changes:
- [x] enabled Url in pre-request scripting

### How to test:

Please run this script and check its output.
```
const { Url, QueryParam, Variable } = require('insomnia-collection');

const url = new Url({
		auth: {
				username: 'usernameValue',
				password: 'passwordValue',
		},
		hash: 'hashValue',
		host: ['hostValue', 'com'],
		path: ['pathLevel1', 'pathLevel2'],
		port: '777',
		protocol: 'https:',
		query: [
				new QueryParam({ key: 'key1', value: 'value1' }),
				new QueryParam({ key: 'key2', value: 'value2' }),
		],
		variables: [
				new Variable({ key: 'varKey', value: 'varValue' }),
		],
});

console.log(url.toString());
```
